cmd/aries-js-worker/main.go

```diff
@@ -104,7 +103,8 @@ func takeFrom(in chan *command) func(js.Value, []js.Value) interface{} {
 
 			return nil
 		}
-		in <- cmd
+
+		go func() { in <- cmd }()
```

Comment:
normally it would be odd to spin a goroutine on every call, but at least this stabilizes the tests until there is a better idea.

Reply:
It can also be fixed with a buffered channel or worker pool. I guess that the problem is with the `pipe` function. I can try with the worker pool.

Reply:
@troyronda Done! I think this is the best we can do.